packages/sdk/src/types/Stackable.ts

```diff
@@ -0,0 +1,80 @@
+import { RewardRedemptionParams } from './Rewards'
+import { GiftRedemptionParams } from './Gift'
+import { OrdersCreateResponse } from './Orders'
+import { ApplicableToResultList } from './ApplicableTo'
+import { DiscountVouchersTypes, DiscountVouchersEffectTypes, DiscountUnitVouchersEffectTypes } from './DiscountVoucher'
+import { SimpleProduct, SimpleSku } from './Products'
+import { LoyaltyPointsTransfer } from './Loyalties'
+
+interface StackableOptionsBase {
+	include_orders?: boolean
+	extended_redemptions?: boolean
+}
+
+export type StackableOptionsValidation = StackableOptionsBase & {
+	expand: ['order'] | ['redeemable'] | ['order', 'redeemable']
+}
+
+export type StackableOptionsRedemption = StackableOptionsBase & {
+	expand: ['order'] | ['order', 'redemption'] | ['redeemable', 'redemption'] | ['redemption']
+}
+
+export type StackableRedeemableObject = 'promotion_stack' | 'promotion_tier' | 'voucher'
+
+export interface StackableRedeemableParams {
+	object: StackableRedeemableObject
+	id: string
+	reward?: RewardRedemptionParams
+	gift?: GiftRedemptionParams
+}
+
+export type StackableRedeemableResponseStatus = 'APPLICABLE' | 'INAPPLICABLE' | 'SKIPPED'
+
+export interface StackableRedeemableResultDiscountUnit {
+	effect: DiscountUnitVouchersEffectTypes
+	unit_off: number
+	unit_type: string
+	sku?: SimpleSku
+	product?: SimpleProduct
+}
+
+export interface StackableRedeemableResultDiscount {
+	type: DiscountVouchersTypes
+	effect: DiscountVouchersEffectTypes
+	amount_off: number
+	percent_off: number
+	amount_limit: number
+	fixed_amount: number
+	unit_off: number
+	unit_type: string
```

Comment:
I think these must be optional (with ?) because in most cases only one of these will be returned
(I didn't test that, just quick look at the code)

Reply:
Thanks, fixed in 355b3d4b4bf41feca16516dc4d606808bc603091